cmd/plume/upload.go

```diff
@@ -126,17 +200,20 @@ func getImageVersion(imagePath string) string {
 	return version
 }
 
-func writeFile(client *http.Client, filename string) error {
+// Write file to Google Storage
+func writeFile(client *http.Client, filename, destname, projec, projectID string) error {
 	ctx := cloud.NewContext(projectID, client)
```

Comment:
Yeah, just seems to be a quick in the cloud library: https://godoc.org/google.golang.org/cloud/storage